server/game/cards/02.2-DD/JoseMorales.js

```diff
@@ -0,0 +1,15 @@
+const DudeCard = require('../../dudecard.js');
+
+class JoseMorales extends DudeCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => this.game.shootout && this.isParticipating(),
+            match: this,
+            effect: ability.effects.modifySkillRating('huckster', this.bullets)
```

Comment:
Did not test this, but I believe this will need dynamic effect. There is no such effect for Skill ratings in `effects.js`. I can put it there and push to the PR.